common/tmx_level.go

```diff
@@ -201,18 +203,10 @@ func pointStringToLines(str string, xOff, yOff float64) []*Line {
 
 		p1 := engo.Point{x1, y1}
 		p2 := engo.Point{x2, y2}
-		newLine := &Line{p1, p2}
+		newLine := &engo.Line{p1, p2}
 
 		lines = append(lines, newLine)
```

Comment:
This isn't correct either. `lines` is already of length `len(floatPts)-1`. Since you're looping over `i`, you should use `lines[i] = newLine`. Using `append` will only grow `lines` even more.